.github/workflows/docker-aws.yml

```
@@ -0,0 +1,152 @@
+          fi
+      
+      - name: Create tag
+        if: ${{ (github.ref  == 'refs/heads/suite-c4t') && (github.event_name == 'push') && (steps.setDefaults.outputs.create_tag == 'true') }}
+        run: |
+          git config --local user.email "github-actions[bot]@users.noreply.github.com"
+          git config --local user.name "github-actions[bot]"
+          git tag -a ${{ steps.setDefaults.outputs.tag }} -m "create tag ${{ steps.setDefaults.outputs.tag }} for staging"
+          git push --tags
+
+      - name: Configure AWS credentials
+        uses: aws-actions/configure-aws-credentials@v2
```

actionlint (1.7.7)

the runner of "aws-actions/configure-aws-credentials@v2" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)